recipes-domu/domu-image-weston/domu-image-weston.bbappend

```diff
@@ -11,9 +11,9 @@ XT_QUIRK_BB_ADD_LAYER += "meta-xt-prod-extra"
 ################################################################################
 # Renesas R-Car
 ################################################################################
-SRC_URI_rcar = "repo://github.com/xen-troops/manifests;protocol=https;branch=vgpu-dev;scmdata=keep"
+SRC_URI_rcar = "repo://github.com/xen-troops/manifests;protocol=https;branch=master;manifest=prod_devel/domu.xml;scmdata=keep"
 XT_QUIRK_PATCH_SRC_URI_rcar = "file://${S}/meta-renesas/meta-rcar-gen3/docs/sample/patch/patch-for-linaro-gcc/0001-rcar-gen3-add-readme-for-building-with-Linaro-Gcc.patch;patchdir=meta-renesas"
-XT_BB_LAYERS_FILE_rcar = "meta-rcar-gen3/doc/bblayers.conf"
+XT_BB_LAYERS_FILE_rcar = "meta-xt-prod-extra/conf/bblayers.conf"
```

Comment:
From [2]: The first thing BitBake does is parse base configuration metadata. Base configuration metadata consists of the bblayers.conf file to determine what layers BitBake needs to recognize, all necessary layer.conf files (one from each layer), and bitbake.conf.
Well, we have to experiment, but my feeling is that bblayers is ok to be there
@iartemenko can you please confirm, that, for example, if we put bad bblayers.conf in one of the layers (except ours) it is not picked up automatically?